TicketSystem/TicketSystem.BLL/Services/TicketService.cs

```diff
@@ -62,7 +62,7 @@ public async Task<Ticket> UpdateTicket(Ticket ticket, CancellationToken cancella
     {
         var ticketEntity = _mapper.Map<TicketEntity>(ticket);
 
-        await _ticketRepository.Update(ticketEntity, cancellationToken);
+        await _ticketRepository.Update(cancellationToken, ticketEntity);
```

Comment:
Looks like the order of the parameters was right :)

Reply:
I expanded this method by adding params TEntity[]

Reply:
Looks ugly, but okay
I suggest to separate `Update `and `UpdateRange `methods in the future